extension/intents/navigation/navigation.toml

```diff
@@ -5,6 +5,7 @@ match = """
   images (of | for) [query] [service=images]
   (do a |) (search | search on | query on | lookup on | look up on | look on | look in | look up in | lookup in) (my |) [service:serviceName] (for | for the |) [query] (for me |)
   (do a |) (search | query ) my [service:serviceName] (for | for the |) [query] (for me|)
+  (do a |) (search | query ) (this site | here) (for | for the |) [query] (for me|)
```

Comment:
It would be good to have `(the site | this site | here)` – in my testing the speech-to-text produces "the site" about as often as "this site" (must be hard to tell the difference).